content/administration/install/deploy.rst

```diff
@@ -381,12 +387,11 @@ notifications.
 The solutions to support livechat/motifications in a WSGI application are:
 
 * Deploy a threaded version of Odoo (instead of a process-based preforking
-  one) and redirect only requests to URLs starting with ``/longpolling/`` to
-  that Odoo, this is the simplest and the longpolling URL can double up as
-  the cron instance.
+  one) and redirect only requests to URLs starting with ``/websocket/`` to
+  that Odoo, this is the simplest and the websocket URL can double up as
```

Comment:
The end of the sentence was lost along the way. LTGM otherwise.
```suggestion
  that Odoo, this is the simplest and the websocket URL can double up as the cron instance.
```

Reply:
Thanks for your review, PR updated :+1: